app/src/main/java/com/kks/nimblesurveyjetpackcompose/ui/presentation/survey/SurveyNPSScreen.kt

```diff
@@ -0,0 +1,112 @@
+package com.kks.nimblesurveyjetpackcompose.ui.presentation.survey
+
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.wrapContentWidth
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.ExperimentalMaterialApi
+import androidx.compose.material.Surface
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.constraintlayout.compose.ConstraintLayout
+import com.kks.nimblesurveyjetpackcompose.R
+import com.kks.nimblesurveyjetpackcompose.model.SurveyAnswer
+import com.kks.nimblesurveyjetpackcompose.ui.theme.White50
+
+private const val START_INDEX = 0
+
+@OptIn(ExperimentalMaterialApi::class)
+@Composable
+fun SurveyNPSQuestion(
```

Comment:
Acronyms should be [treated as words](https://nimblehq.co/compass/development/code-conventions/kotlin#naming) 😉 

![Screenshot 2565-09-28 at 14 42 12](https://user-images.githubusercontent.com/53168251/192719121-0cb6bcc5-6e72-4d55-9fdd-426f662f94e4.png)

```suggestion
fun SurveyNpsQuestion(
```
Shall we rename the file to `SurveyNpsQuestion.kt` as well?